packages/block-editor/src/components/inserter/style.scss

```diff
@@ -28,12 +28,24 @@ $block-inserter-search-height: 38px;
 .block-editor-inserter__toggle {
 	display: inline-flex;
 	align-items: center;
-	background: none;
 	cursor: pointer;
 	border: none;
 	outline: none;
+	padding: 0;
 	transition: color 0.2s ease;
+
+	// Spot color button.
+	// @todo: This button should ideally be 32px.
```

Comment:
The inserter is used in different places and not just the top toolbar. I think the "blue button" shouldn't be solved using CSS but using the `isPrimary` prop in JS and only for the top toolbar inserter.